sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -389,8 +388,8 @@ export class CertificateClient {
         maxresults: continuationState.maxPageSize,
         ...options,
       };
-      const currentSetResponse = await withTrace(
-        "listPropertiesOfCertificateVersions",
+      const currentSetResponse = await tracingClient.withSpan(
+        "CertificateClient.listPropertiesOfCertificateVersions",
```

Comment:
I see that in other "Page" operations you are suffixing the span name with Page, but not in this particular clase. Is this function special?

Reply:
Thanks, updated!